src/languages/en.ts

```diff
@@ -4067,6 +4091,12 @@ const translations = {
                 description: `Rules run in the background and keep your spend under control so you don't have to sweat the small stuff.\n\nRequire expense details like receipts and descriptions, set limits and defaults, and automate approvals and payments – all in one place.`,
                 onlyAvailableOnPlan: 'Rules are only available on the Control plan, starting at ',
             },
+            perDiem: {
+                title: 'Per diem',
+                description:
+                    'Per diem is a great way to keep your daily costs compliant and predictable whenever your employees travel. Enjoy features like custom rates, default categories, and more granular details like destinations and subrates.',
+                onlyAvailableOnPlan: 'Per diem are only available on the Control plan, starting at ',
```

Comment:
```suggestion
                onlyAvailableOnPlan: 'Per diem is only available on the Control plan, starting at ',
```

NIT: Google say plural for per diem is per diems.

So, this in `onlyAvailableOnPlan` might need to be confirmed if this should be "Per diems are" or "Per diem is" or retain as it is.